src/main.cc

```diff
@@ -1824,6 +1835,10 @@ Parse(TRITONSERVER_ServerOptions** server_options, int argc, char** argv)
       TRITONSERVER_ServerOptionsSetBufferManagerThreadCount(
           loptions, std::max(0, buffer_manager_thread_count)),
       "setting buffer manager thread count");
+  FAIL_IF_ERR(
+      TRITONSERVER_ServerOptionsSetModelLoadThreadCount(
+          loptions, std::max(1u, model_load_thread_count)),
+      "setting model load thread count");
```

Comment:
This question is a duplicate of [this](https://github.com/triton-inference-server/core/pull/86#discussion_r892926372). Question: Since this is a user provided variable, should we set an upper bound or is that being too opinionated? The user could potentially set thread count to -1 and lock up the system by generating a large number of threads. What would be a reasonable max here, I'm thinking 2x or 3x of `std::thread::hardware_concurrency()`